projects/github.com/nvm-sh/nvm/package.yml

```diff
@@ -0,0 +1,48 @@
+distributable:
+  url: https://github.com/nvm-sh/nvm/archive/v{{version}}.tar.gz
+  strip-components: 1
+versions:
+  github: nvm-sh/nvm
+warnings:
+  - vendored
+runtime:
+  env:
+    NVM_DIR: "{{prefix}}"
+dependencies:
+  linux:
+    curl.se: '*'
+build:
+  script:
+    - run: mkdir -p bin libexec etc/bash_completion.d
+      working-directory: "{{prefix}}"
+    - install nvm.sh nvm-exec {{prefix}}/libexec/
+    - install bash_completion {{prefix}}/etc/bash_completion.d/nvm
+    - run: |
+        ln -s ../libexec/nvm-exec || true
+      working-directory: "{{prefix}}/bin"
+    - run: |
+        sed -i.bak "s|unset NVM_CD_FLAGS|unset NVM_CD_FLAGS \&\& DIR=\$DIR/../libexec|g" nvm-exec
+        rm *.bak
+      working-directory: "{{prefix}}/libexec"
+provides:
+  - bin/nvm-exec
+test:
+  script:
+    - NODE_VERSION=pkgx nvm-exec || echo $? | grep 127
```

Comment:
what is this doing?

Reply:
I looked it up in `homebrew`
As far as I understand this way we just check if `nvm-exec` starts and if it can find `nvm.sh`

```
test do
    output = pipe_output("NODE_VERSION=homebrewtest #{prefix}/nvm-exec 2>&1")
    refute_match(/No such file or directory/, output)
    refute_match(/nvm: command not found/, output)
    assert_match "N/A: version \"homebrewtest\" is not yet installed", output
  end
```

Reply:
oh, right. my brain inserted quotes that didn't exist.

Reply:
i think testing the error condition is fine.